tests/installation/installation_overview.pm

```diff
@@ -69,6 +69,18 @@ sub run {
         $self->deal_with_dependency_issues;
         assert_screen "inst-xen-pattern" if get_var('XEN');
         ensure_ssh_unblocked;
+        # Check the systemd target, see poo#45020
+        if (get_var('DESKTOP')) {
+            # QAM do some installs with DESKTOP=textmode, but install gnome... 
+            # and switching between ttys does not work for hyperv.
+            return if (get_var('QAM_MINIMAL') || check_var('MACHINE', 'svirt-hyperv'));
+            my $target = check_var('DESKTOP', 'textmode') ? "multi-user" : "graphical";
+            select_console 'install-shell';
+            # The default.target is not yet linked, so we have to parse the logs.
+            assert_script_run("grep 'target has been set' /var/log/YaST2/y2log |tail -1 |grep \"$target\"", fail_message => "Wrong systemd target detected, aborting");
+            select_console 'installation';
+        # The follo
```

Comment:
some orphan comment.

Reply:
Thanks anyway, I had forgotten this one :)